# Bans comments (ts-template/example-rule)

This is example rule.

## 📖 Rule Details

Bans comments from being used.

👍 Examples of **correct** code for this rule:

```ts
const a = 1;
```

👎 Examples of **incorrect** code for this rule:

```ts
// comment
const a = 1;
```
