# Bans comments (ts-template/example-rule)

This is example rule.

## ğŸ“– Rule Details

Bans comments from being used.

ğŸ‘ Examples of **correct** code for this rule:

```ts
const a = 1;
```

ğŸ‘ Examples of **incorrect** code for this rule:

```ts
// comment
const a = 1;
```
